File: Python/D10 - T/3 - pattern/3.py
# Write a program to print following pattern.
#         1
#       2 2
#     3 3 3
#   4 4 4 4
# 5 5 5 5 5

count = 5
for i in range(1, 6):
    for k in range(5,i,-1):
        print(" ", end=" ")
    for j in range(1, i+1):
        print(i, end=" ")
    print()

File: C/D6/3/3.cpp
// Write a C program to read marks from keyboard & display equivalent grade
// according to following table.
// 100-80 = distinction
// 79-60 = first class
// 59-40 = second class

#include <stdio.h>

int main(){
    int marks;

    printf("Enter marks: ");
    scanf("%d", &marks);

    if(marks < 101 ){
        if(marks >= 80 && marks <= 100){
            printf("Grade: Distinction");
        }
        else if(marks >= 60 && marks <= 79){
            printf("Grade: First Class");
        }
        else if(marks >= 40 && marks <= 59){
            printf("Grade: Second Class");
        }
    }
    else{
        printf("Enter should not be more than 100");
    }
}